merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/remotes/postgres/InsertActivityTypesAction.java

```diff
@@ -50,12 +51,21 @@ public void apply(final int modelId, Collection<ActivityType> activityTypes, Map
 
 
         if (activityType.subsystem().isPresent()) {
-          int subsystemIndex = mapSubsystemsToIds.get(activityType.subsystem().get());
+          Integer subsystemIndex = mapSubsystemsToIds.get(activityType.subsystem().get());
+          if (subsystemIndex == null) {
+            throw new FailedInsertException("Unknown subsystem '" + activityType.subsystem().get() + "' for model_id=" + modelId + " in merlin.activity_type");
+          }
```

Comment:
A more standard way of doing this check is:
```suggestion
if(!mapSubsystemsToIds.containsKey(activityType.subsystem().get())) {
  throw new FailedInsertException("Unknown subsystem '" + activityType.subsystem().get() + "' for model_id=" + modelId + " in merlin.activity_type");
}
int subsystemIndex = mapSubsystemsToIds.get(activityType.subsystem().get());
```
This makes it clear that:
1. the exception is being thrown because the key is not in the map **and** 
2. that `null` is not an accepted `value` for a `key` in the map.